caffeine@patapon.info/extension.js

```diff
@@ -903,13 +931,13 @@ class Caffeine extends QuickSettings.SystemIndicator {
 
     _appWorkspaceChanged() {
         // Reset signal for Add/remove windows on workspace
-        if (this._appAddWindowSignalId > 0) {
```

Comment:
In the first place, I bug when I saw my old code because I didn't understand why I chose to compare to 0. Then I remember that using "null" value was causing log error "invalid (NULL) pointer instance" or something like that. 

So I just ran few tests with your changes and this time there is no more errors in the logs when switching workspace:  maybe because of a change/fix upstream since I wrote this code, or maybe I just miss something :-). Anyway, it's working fine this time and it's better like this, thanks.

Reply:
Strange, I've never run into anything like that before, I just changed this to be a bit more literal, since we're really checking that it's for a signal, not specifically that it's above 0.